library/default-synths-extra.scd

```diff
@@ -294,3 +294,120 @@
 	}).add
 );
 
+
+// tuning fork
+// from my own experimentation and from "On the acoustics of tuning forks", Rossing Russell and Brown
+(SynthDef(\superfork, {|out, sustain=1, pan, freq, accelerate=0|
+	var sound, env;
+	freq = freq * Line.kr(1, 1+accelerate, sustain);
+	env = EnvGen.ar(Env.linen(0.001, 0.998, 0.001), timeScale:sustain, doneAction:2);
+	sound = 0.2 * SinOsc.ar([1.0, 6.1, 16.7, 20.1] * freq) * XLine.ar([0.3, 1, 1.3, 2], [1e-4, 1e-7, 1e-6, 1e-7], sustain*[4,4,0.5,0.333]);
+	OffsetOut.ar(out, DirtPan.ar(sound, ~dirt.numChannels, pan, env));
+}).add);
+
+
+// hammond B3 sim
+// freq adjustments courtesy Tom Wiltshire (electricdruid.net)
+// drawbar settings selectable with "voice" and from many public domain sources,
+// Google "Hammond drawbar settings" and add your own in the big SelectX block
+// "perc" "percf" and "decay" are an attempt at the percussion, no idea if it sounds at all reasonable
+// vintage Hammonds had percf as 2 or 3 (switchable), two perc levels (maybe roughly 0.7 and 1.2?),
+// and two decay options (roughly 0 and maybe 1ish?)
+// "vibrato", "vrate", "perc", "percf" are all new params you'll need to define in Tidal if you want to change them
+(SynthDef(\superhammond, {|out, sustain=1, decay=0, pan, freq, vibrato=0.5, vrate=7,
+	voice=0, perc=0, percf=2 |
+	var freqs, amps, sound, click, env, ampscale, attrel;
+	attrel = (sustain/2).clip(0,0.005);
+	env = EnvGen.ar(Env.linen(attrel, sustain-(2*attrel), attrel), doneAction:2);
+	freqs = [0.5, 1.4988, 1, 2, 2.9976, 4, 5.0409, 5.9953, 8] * freq;
+	freqs = 2**(freqs < 63) * freqs; // fold up bottom register
+	freqs = freqs / (2**(freqs > 6000)); // fold down high harmonics
+	freqs = freqs / (2**(freqs > 6000)); // do it again for the really high notes
+	amps = SelectX.kr(voice, [
```

Comment:
you could write `#[ ` (just once before the outermost array, to minimize memory impact. Not a big thing, but still worth it.